### PR TITLE
Fix content-type, allow all directives, force Contact:, allow multiple same directives

### PR DESCRIPTION
**Pull request by joker314**
There were a few issues I observed which made the generated security.txt file not too useful, or not compliant with the specification.

- **The resulting file was of Content-Type `text/html`, not `text/plain`.** 
  - Harder to read
  - Not spec-compliant
- **Not all directives were allowed**.
  - Only three directives were allowed
  - This PR makes all (including invalid directives) legal
- **No error was generated for lack of a `Contact:`**.
  - This pull request will throw if you fail to provide a `Contact` property (regardless of capitalisation, per spec)
- **Directives were only allowed once.**
  - Multiple contact methods, etc. could not be communicated
  - You can now pass arrays
  - The passing of arrays is now documented in the README
- **The entire file was re-generated every time**.
  - The security.txt file is mostly calculated upon its definition now, and then it's just returned. This is more efficient if it's being requested many times; and allows it to error immediately if the security.txt file is invalid.

### CODE REVIEW

**Comment by gergelyke** [comment]
hi @joker314! thanks a lot for the contribution! I've just added travis & linting to make sure we have consistent code style across the project. can you please update your PR?

**Comment by joker314** [comment]
Err, sure. I'm going to change a test, though:

```javascript
t.equal(res.text, `Contact: email@example.com\n`)
```

It's inconsistent, because the other test does not require the final line to be terminated by newline.

**Comment by joker314** [comment]
🎉

File: index.js [hunk 1]
```diff
@@ -1,13 +1,49 @@
...
-    response.send(resp)
+/**
+ * Internal function. Returns arrays as passed, but converts some
+ * string X into an array [X] of length 1. For example, "abc" would
+ * become ["abc"].
+ *
+ * @param {string|array} stringOrArray - The string or array to force to an array
+ * @return {array} The array form of the argument
+ */
+function forceToArray (stringOrArray) {
```
Comment (gergelyke) [inline index.js]: nit: can we pick a more descriptive name, like `ensureArray`?

**Review by gergelyke — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (joker314) [reply index.js]: Sure!

**Comment by gergelyke** [comment]
published under `2.0.0`

**Pull request merged**